staxapp/api.py

```diff
@@ -13,13 +13,7 @@ class Api:
     def _headers(cls, custom_headers) -> dict:
         headers = {
             **custom_headers,
-            "User-Agent": json.dumps(
-                {
-                    "platform": Config.platform,
-                    "python_version": Config.python_version,
-                    "sdk_version": Config.sdk_version,
-                }
-            ),
+            "User-Agent": f"platform/{Config.platform} python/{Config.python_version} sdk/{Config.sdk_version}",
```

Comment:
```suggestion
 "User-Agent": f"platform/{Config.platform} python/{Config.python_version} staxapp/{Config.sdk_version}",
```

Reply:
small change to meet rfc7231#section-5.5.3 (in my opinion only)

Reply:
Done